Doc/whatsnew/3.14.rst

```diff
@@ -222,6 +222,14 @@ pickle
   For more details, please see :ref:`pickle protocols <pickle-protocols>`.
 
 
+multiprocessing
```

Comment:
Please, order the module's entry alphabetically. This section should be placed between the `json` and `operator` sections above.